src/client.rs

```diff
@@ -362,6 +368,19 @@ impl Client {
         .await
     }
 
+    /// Creates a persistent subscription group on a the $all stream.
+    pub async fn create_persistent_subscription_to_all<GroupName>(
+        &self,
+        group_name: GroupName,
+        options: &PersistentSubscriptionToAllOptions,
```

Comment:
What's the difference with the PersistentSubscrptionOption ?
int the .net Client the options for create & createToAll are the same , not a different structure

Reply:
Ah ok I see , it